README.md

```diff
@@ -23,6 +23,13 @@ git clone https://github.com/TabbyML/quick-question.git
 # Save your OPENAI_API_KEY to file as secret
 echo YOUR_OPENAI_API_KEY > openai_api_key.txt
 
+# Setup your own codebase (Optional).
+# Replace 'your_project_name' and 'your_github_repository_name' with your own,
+# 'your_github_repository_name' could be something like 'TabbyML/quick-question'.
+# See also ./data/diffusers/metadata.json
+mkdir ./data/your_project_name
+echo '{"name":"your_github_repository_name","exampleQueries":["How to ...?"]}' > ./data/your_project_name/metadata.json
```

Comment:
`exampleQueries` is required here.  
Should we set a placeholder in code instead of in metadata.json?

Reply:
I don't understand.

You can still have exampleQueries in an example with content of `metadata.json` file in markdown.